Dockerfile

```diff
@@ -1,7 +1,7 @@
-FROM ruby:2.3-alpine
+FROM ruby:2.4-alpine
 
 # https://github.com/gliderlabs/docker-alpine/issues/53#issuecomment-125671731
-RUN apk add --update build-base libxml2-dev libxslt-dev
+RUN apk add --update build-base libxml2-dev libxslt-dev libcurl
```

Comment:
Ah, nevermind, just saw the commit message for b852a8605e91587bb2596fc7828403764fc3afb3.